core/circuitbreaker/rule_manager.go

```diff
@@ -16,11 +16,11 @@ type CircuitBreakerGenFunc func(r *Rule, reuseStat interface{}) (CircuitBreaker,
 var (
 	cbGenFuncMap = make(map[Strategy]CircuitBreakerGenFunc, 4)
 
-	breakerRules = make(map[string][]*Rule)
-	breakers     = make(map[string][]CircuitBreaker)
-	updateMux    = &sync.RWMutex{}
-	currentRules = make([]*Rule, 0)
-
+	BreakerRules         = make(map[string][]*Rule)
```

Comment:
Why change the scope of a variable? It's not safe

Reply:
I agree with you, but can modify a variable through function, I will modify it.